hazelcast/src/test/java/com/hazelcast/internal/util/AbstractClockTest.java

```diff
@@ -74,6 +89,25 @@ protected void startIsolatedNode() {
         }
     }
 
+    private void randomizeMulticastPort(Object config) throws Exception {
+        // By default auto detection uses multicast.
+        // See Node.shouldUseMulticastJoiner().
+
+        Method getNetworkConfig = config.getClass().getDeclaredMethod("getNetworkConfig");
```

Comment:
Because this is a little strange test. My understanding is the following:

1. We need to modify a static final field, so in this test we copy all hazelcast classes via reflection.
2. So if you want to interact with them, you need to use those copied classes.
3. So for example Config class in junit test and config class in this new class loader are different classes.

I tried using getters and setters as well but I get an exception. Above is my understanding of that.